doc/_templates/layout.html

```diff
@@ -2,7 +2,7 @@
 
 {% block menu %}
   {{ super() }}
-  <a href="cabal-projectindex.html">Reference</a>
-  <a href="genindex.html">Index</a>
+  <a href="cabal-projectindex.html">Quicklinks</a>
+  <a href="genindex.html">Word index</a>
```

Comment:
I associate “Word index” to Forth. “Index” is

```
A table for facilitating reference to topics, names, and
the like, in a book, usually giving the page on which a
particular word or topic may be found; -- usually
alphabetical in arrangement, and printed at the end of the
volume. Typically found only in non-fiction books.
[1913 Webster +PJC]
```

what does “word” add to it?

Reply:
Indeed, reverted.